sklearn/metrics/cluster/_bicluster.py

```diff
@@ -57,8 +57,8 @@ def _pairwise_similarity(a, b, similarity):
 def consensus_score(a, b, *, similarity="jaccard"):
     """The similarity of two sets of biclusters.
 
-    Similarity between individual biclusters is computed. Then the
-    best matching between sets is found using the Hungarian algorithm.
+    Similarity between individual biclusters is computed. Then the best
+    matching between sets is found using a modified Jonker-Volgenant algorithm.
```

Comment:
Let's make it explicit that computing the consensus score can be reduced to solving a linear sum assignment problem and that the Jonker-Volgenant algorithm is one way to do it (to make the see also reference more relatable).

```suggestion
    Similarity between individual biclusters is computed. Then the best
    matching between sets is found by solving a linear sum assignment problem
    using a modified Jonker-Volgenant algorithm.
```

Reply:
Agreed